tests/apollo/util/bft.py

```diff
@@ -146,7 +145,8 @@ async def wrapper(*args, **kwargs):
                               f'with n={config.n}, f={config.f}, c={config.c}, '
                               f'num_clients={config.num_clients}, '
                               f'num_ro_replicas={config.num_ro_replicas}')
-                        await async_fn(*args, **kwargs, bft_network=bft_network)
+                        with log.start_task(action_type=f"{bft_network.current_test}_num_clients={config.num_clients}"):
```

Comment:
I missed this. Let's remove the `_num_clients`, and instead give the n, f and c values.

Reply:
n, f, c are part of bft_network.current_test.
This log line looks like the following:
{"action_status": "started", "timestamp": 1600760070.9049213, "task_uuid": "9cf8f66d-0e35-40b9-a6fb-055373d61ca2", "action_type": "**test_auto_vc_all_nodes_up_no_requests_n=4_f=1_c=0_num_clients=30**", "task_level": [1]}

Reply:
Then it's perfect, thanks Oded! No more comments from my side.

You can merge into master.

Reply:
Great, thanks!